app/Main.hs

```diff
@@ -7,5 +7,5 @@ import qualified Utils (run_day, readInputFile)
 main :: IO ()
 main = do
     Utils.run_day (Utils.readInputFile "real_data/d1.txt") Day1.run;
-	Utils.run_day (Utils.readInputFile "real_data/d2.txt") Day2.run
+	Utils.run_day (Utils.readInputFile "real_data/d2.txt") Day2.run;
```

Comment:
You generally always should prefer whitespace alignment over semicolons.

Reply:
Can you elaborate on this? Removing the semicolon on line 9 results in an error, should I only be removing it on line 10?

Reply:
That's because your whitespace isn't aligned properly. Semicolons are an alternative to statement delimitation over whitespace. See https://en.wikibooks.org/wiki/Haskell/Indentation.